src/checkout_body/checkout_body.css

```
@@ -0,0 +1,207 @@
+.checkout_section {
+    padding: 86px 174px 44px 175px;
+    display: flex;
+    justify-content: space-between;
+}
+
+.checkout_header {
+    font-size: 36px;
+    font-weight: 600;
+    letter-spacing: .5px;
+}
+
+.checkout_form {
+    padding-top: 36px;
+}
+
+.double_iputs_container {
```

```suggestion
.double_inputs_container {
```
please, be more attentive to spelling. you make a lot of these mistakes/typos. use a spellchecker or smth